src/ParallelAlgorithms/Amr/Actions/Initialize.hpp

```diff
@@ -14,37 +13,49 @@
 #include "Utilities/MakeArray.hpp"
 #include "Utilities/TMPL.hpp"
 
-/// \cond
-template <size_t VolumeDim>
-class ElementId;
-/// \endcond
-
 namespace amr::Initialization {
-/*!
- * \brief %Mutator meant to be used with
- *  `Initialization::Actions::AddSimpleTags`to initialize flags used for
- * adaptive mesh refinement
- *
- * DataBox:
- * - Adds:
- *   - `amr::Tags::Flags<Dim>`
- *   - `amr::Tags::NeighborFlags<Dim>`
- * - Removes: nothing
- * - Modifies: nothing
- *
- */
+/// \ingroup InitializationGroup
+/// \brief Initialize items related to adaptive mesh refinement
+///
+/// \details See the type aliases defined below for what items are added to the
+/// GlobalCache, MutableGlobalCache, and DataBox and how they are initialized
 template <size_t Dim>
 struct Initialize {
-  using return_tags =
-      tmpl::list<amr::Tags::Flags<Dim>, amr::Tags::NeighborFlags<Dim>>;
+  /// Tags for constant items added to the GlobalCache.  These items are
+  /// initialized from input file options.
```

Comment:
I have no objection to someone else doing that...

Reply:
In this PR it would be enough to (1) delete these comments that we shouldn't (have to) duplicate in every initializer, and (2) link to `InitializeItems` in the docs (that's super important, else it's unclear what all these aliases even mean and why they're there). I opened #4761 for the rest.

Not sure what you mean with "no objections to someone else doing that". In my opinion we can't expect others to resolve issues like this. Happy to discuss in a call if you like.